pkg/installer/daemonset.go

```diff
@@ -146,7 +146,7 @@ func createDaemonSet(ctx context.Context, c *Config) error {
 						args = append(args, "--loopback-only")
 					}
 					if c.DynamicDriveDiscovery {
-						args = append(args, "--dynamic-drive-discovery")
+						args = append(args, "--enable-dynamic-discovery")
```

Comment:
should we support backward compatility here? renaming this flag will break backward compatibility. 

If v2.0.1 client is used to install, `--dynamic-drive-discovery` will be set on the binary.

Reply:
yeah, lets keep the existing flag